cliquet_docs/reference/configuration.rst

```diff
@@ -73,6 +73,22 @@ following setting should be declared in the ``.ini`` file:
     cliquet.record_mushroom_patch_enabled = false
 
 
+Setting the service in read_only mode
+:::::::::::::::::::::::::::::::::::::
+
+It is also possible to deploy a cliquet service in read_only mode.
+
+Instead of listing all the endpoint, you can use the ``read_only`` settings::
```

Comment:
of having to specify every resources endpoint and verbs, the `<read only>` setting allows to make sure the services only allows read operations.